Applications/PlazaDetails/Strength.qmd

```diff
@@ -1,27 +1,32 @@
+The following are among the popular services available in OpenEO Algorithm Plaza. Here, we have presented an average credit strength for these services. Please note that the credits can vary slightly depending on the time interval and area of interest.
 
 The purpose of this document is to solely provide users with an assumption on how these credits work and their strength for varying services.
 
 ##### NDVI
-For the calculation of Normalized Difference Vegetation Index (NDVI), you can acquire ~250 NDVI images, each with an area of 1 hectare, i.e. 1 NDVI image patch of 1 hectare cost 3-5 credits.
+
+The Normalized Difference Vegetation Index (NDVI) is a key indicator of vegetation health. It is calculated by comparing the difference between near-infrared (which vegetation strongly reflects) and red light (which vegetation absorbs) in satellite imagery.
+Calculating the NDVI of a square hectare costs 3-5 credits per hectare. 
 
 ##### NDII
-Computation of the Normalized Difference Infrared Index (NDII) provides the user with an indication of the water content in the plant canopies. A straightforward calculation of the NDII can be achieved from openEO algorithm plaza NDII service with the use of 4-8 credits per hectare area, i.e. you can use your 1000 credits for computing NDII in a given area of ~117 hectares.
+
+The Normalized Difference Infrared Index (NDII) computation offers insight into plant canopies' water content. Using the openEO Algorithm Plaza, NDII service costs 4-8 credits per hectare.
 
 ##### Biomass
-The Biomass service provided by openEO algorithm plaza can be a critical component in tracking and quantifying carbon capture in agriculture and natural vegetation. Users can use this service with approximately 20-40 credits per hectare, i.e. ~33-hectare area with a total of 1000 credits.
+
+The Biomass service provided by the openEO Algorithm Plaza is a critical component in tracking and quantifying carbon capture in agriculture and natural vegetation. Users can use this service with approximately 20-40 credits per hectare.
 
 ##### CropSAR
-The operational agricultural monitoring can be a key use case for this service as it uses Sentinel-1 radar observations to augment those of Sentinel-2. By doing so, more continuous vegetation properties monitoring can be performed, including vegetation information for situations where these properties cannot be retrieved from Sentinel-2 observations due to cloud occurrence. Users can use this service with approximately 40-60 credits per hectare, i.e. ~20-hectare area with a total of 1000 credits.
+
+The CropSAR service is a key service for monitoring agricultural conditions. An AI model which fills missing Sentinel-2 observations due to clouds based on Sentinel-1 radar observations. By doing so, more continuous vegetation properties monitoring can be performed. Users can use this service with approximately 40-60 credits per hectare.
```

Comment:
did we validate this cost?

Reply:
No, this was based on manual check done more than a year ago. We included it have simply to provide with an idea. But I think it is nice to rethink if we want to keep it.